src/main/java/com/github/dockerjava/api/model/Ports.java

```diff
@@ -70,7 +71,7 @@ public void bind(ExposedPort exposedPort, Binding binding) {
             if (binding == null) {
                 ports.put(exposedPort, null);
             } else {
-                ports.put(exposedPort, new Binding[] { binding });
+                ports.put(exposedPort, new Binding[]{binding});
```

Comment:
Probably src style may exist, but at least i synced into one style all this braces.